### ISSUE TITLE
TypeError: '>' not supported between instances of 'list' and 'int'

### ISSUE THREAD

**Issue by Billybangleballs** [bkerler/mtkclient#760]
```
user@home:~/mtkclient $ python3 mtk plstage --preloader=preloader_no_hdr.bin
MTK Flash/Exploit Client V1.6.3 (c) B.Kerler 2018-2023

Preloader - Status: Waiting for PreLoader VCOM, please connect mobile

Port - Hint:

Power off the phone before connecting.
For brom mode, press and hold vol up, vol dwn, or all hw buttons and connect usb.
For preloader mode, don't press any hw button and connect usb.
If it is already connected and on, hold power for 10 seconds to reset.


...........

Port - Hint:

Power off the phone before connecting.
For brom mode, press and hold vol up, vol dwn, or all hw buttons and connect usb.
For preloader mode, don't press any hw button and connect usb.
If it is already connected and on, hold power for 10 seconds to reset.


.....Port - Device detected :)
Preloader -     CPU:                    MT8163()
Preloader -     HW version:             0x0
Preloader -     WDT:                    0x10007000
Preloader -     Uart:                   0x11002000
Preloader -     Brom payload addr:      0x100a00
Preloader -     DA payload addr:        0x201000
Preloader -     CQ_DMA addr:            0x10212c00
Preloader -     Var1:                   0xb1
Preloader - Disabling Watchdog...
Traceback (most recent call last):
  File "mtk", line 855, in <module>
    mtk = Main(args).run(parser)
  File "/home/user/mtkclient/mtkclient/Library/mtk_main.py", line 475, in run
    if mtk.preloader.init():
  File "/home/user/mtkclient/mtkclient/Library/mtk_preloader.py", line 197, in init
    self.setreg_disablewatchdogtimer(self.config.hwcode)  # D4
  File "/home/user/mtkclient/mtkclient/Library/mtk_preloader.py", line 434, in setreg_disablewatchdogtimer
    res = self.write32(addr, value)
  File "/home/user/mtkclient/mtkclient/Library/mtk_preloader.py", line 296, in write32
    return self.write(addr, dwords, 32)
  File "/home/user/mtkclient/mtkclient/Library/mtk_preloader.py", line 274, in write
    if status > 0xFF:
TypeError: '>' not supported between instances of 'list' and 'int'
user@home:~/mtkclient $

```

**Comment by Billybangleballs** [comment]
An official stale bug.

**Comment by bkerler** [comment]
Should be fixed with commit [a53a9ef](https://github.com/bkerler/mtkclient/commit/a53a9ef73bb21239d1b6150ed32e2f5474e46e38). This won't fix the issue why a read timeout happens however.
